### PR TITLE
Use heredoc with credentials template

### PR DESCRIPTION
**Pull request by yhirano55**
### Summary

* Use heredoc with credentials template.
* Fix indentation for aws config

### CODE REVIEW

File: railties/lib/rails/generators/rails/credentials/credentials_generator.rb [hunk 1]
```diff
@@ -42,9 +42,14 @@ def credentials
         end
 
         def credentials_template
-          "# aws:\n#  access_key_id: 123\n#  secret_access_key: 345\n\n" +
-          "# Used as the base secret for all MessageVerifiers in Rails, including the one protecting cookies.\n" +
-          "secret_key_base: #{SecureRandom.hex(64)}"
+          <<-YAML.strip_heredoc
```
Comment (y-yagi) [inline railties/lib/rails/generators/rails/credentials/credentials_generator.rb]: Please add `require "active_support/core_ext/string/strip"`.

Comment (y-yagi) [reply railties/lib/rails/generators/rails/credentials/credentials_generator.rb]: `encrypted` command also has the same issue. Can you fix?
https://github.com/rails/rails/blob/52c9af6c51fbf686762ad2330f36fe9f8ce3c5d7/railties/lib/rails/generators/rails/encrypted_file/encrypted_file_generator.rb#L18

Comment (yhirano55) [reply railties/lib/rails/generators/rails/credentials/credentials_generator.rb]: @y-yagi Thanks for the comments. I've fixed them based on your advices.

**Review by y-yagi — APPROVED** [review]

**Comment by y-yagi** [comment]
Thanks!

**Comment by kaspth** [comment]
@yhirano55 thanks!

@y-yagi don't forget to backport to 5-2-stable 😊

**Comment by y-yagi** [comment]
:+1: Backported: d1e3302fa8dba0da4acdfd36beee55434e226743

**Pull request merged**